src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -134,3 +149,21 @@ $table-cell-height-half: $table-cell-height / 2;
   color: $foreground-2;
   cursor: default;
 }
+
+.kd-replicasetdetail-no-events {
+  margin-top: 4 * $baseline-grid;
+  text-align: center;
+}
+
+md-icon {
+  &.kd-replicasetdetail-no-events-icon {
+    color: $muted;
+    font-size: 1.5 * $headline-font-size-base;
```

Comment:
Without specyfing font-size it will be the same as $headline-font-size-base (first picture).

On the first picture both text and icon are 24px.
On second icon is 36px and text 24px.